guacamole-ext/src/main/java/org/apache/guacamole/properties/FileGuacamoleProperty.java

```diff
@@ -23,13 +23,22 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.regex.Pattern;
 import org.apache.guacamole.GuacamoleException;
 
 /**
  * A GuacamoleProperty whose value is a filename.
  */
 public abstract class FileGuacamoleProperty implements GuacamoleProperty<File> {
 
+    /**
+     * A pattern which matches against the delimiters between values. This is
+     * currently simply a semicolon and any following whitespace. Parts of the
```

Comment:
> ... This is currently simply a semicolon and any following whitespace ...

This needs to be updated to include the comma among the characters used as delimiters (same for other comments).

Reply:
Updated!